source/read/change-streams.txt

```diff
@@ -0,0 +1,253 @@
+.. _ruby-change-streams:
+
+====================
+Monitor Data Changes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: watch, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use a **change stream** to monitor real-time
+changes to your data. A change stream is a {+mdb-server+} feature that
+allows your application to subscribe to data changes on a collection, database,
+or deployment.
+
+When using the {+driver-short+}, you can call the ``watch`` method to return a
+``Mongo::Collection::View::ChangeStream`` object. Then, you can convert the change
+stream to an ``Enumerator`` object and iterate through its content to monitor data
+changes, such as updates, insertions, and deletions.
```

Comment:
There's no need to convert the change stream to an enumerator, as the stream is already iterable. Maybe something like the following?

```suggestion
When using the {+driver-short+}, you can call the ``watch`` method to return a
``Mongo::Collection::View::ChangeStream`` object. Then, you can iterate through
its content to monitor data changes, such as updates, insertions, and deletions.
```

Reply:
I like this wording, updated! Sorry about that confusion